dataflux_core/performance_tests/list_and_download.py

```diff
@@ -83,7 +83,7 @@ def run_list(self, config):
             )
         return list_result
 
-    def run_download(self, config, list_result):
+    def run_download(self, config, list_result, segmented=False):
```

Comment:
Should segmented be part of config?

Reply:
I realize i can just set segmented to be automatic for any expected data set larger than 50 gigs, so i'll update it to trigger off of the existing config.